docs/src/app/docs/layout.tsx

```diff
@@ -60,15 +60,7 @@ export default async function RootLayout({
         <div className="lg:pl-60 pt-24 flex-1">
           <div className="py-10 pl-12 ">
             {/** INNER CONTENT (markdoc goes here) */}
-            <div className="grid gap-6 grid-cols-[auto,12rem]">
-              <div>{children}</div>
-
-              <div className="sticky top-28 self-start">
-                <h5 className="text-xs uppercase text-stone-600">
-                  On this page
-                </h5>
-              </div>
-            </div>
+            <div className="grid gap-6 grid-cols-[auto,12rem]">{children}</div>
```

Comment:
Should we move this grid to the `[slug]/page.tsx` where the `<TableOfContents />` is used? Just thinking about that `12rem` only being used for the ToC so maybe they should stay together rather than being used e.g. on the `/docs` page?

Reply:
Was trying to minimise the merge strife when Johan merges his PR, but you are right, may as well do it properly!
- [ ] Grid to page